SlicingDice.cpp

```diff
@@ -1,22 +1,32 @@
 #include "SlicingDice.h"
 
 
-SlicingDice::SlicingDice(const char* apiUserKey) {
-    host = "api.slicingdice.com/v1";
+SlicingDice::SlicingDice(String apiUserKey) {
+    host = "api.slicingdice.com";
     port = 80;
     apiKey = apiUserKey;
+    test = 0;
```

Comment:
use boolean
find better name

'useProductionCluster'
or something like this.